modules/gapi/test/gapi_statefulkernel_tests.cpp

```diff
@@ -0,0 +1,201 @@
+// This file is part of OpenCV project.
```

Comment:
This test must come to the OCV (CPU) backend directory.

Also please change `statefulkernel` to `stateful_kernel`

Reply:
Moved to cpu directory and renamed to gapi_ocv_kernel_st_tests.cpp ((name is derived from macro name for OCV Stateful kernel).

Reply:
File names should be descriptive. The point of _ST in the macro was that people have to type it when write their kernels. File names are more read than written.